python/apps/taiga/src/taiga/users/api/__init__.py

```diff
@@ -178,6 +184,41 @@ async def update_my_user(request: Request, form: UpdateUserValidator) -> User:
     )
 
 
+#####################################################################
+# delete user
+#####################################################################
+
+
+# TODO
+
+
+#####################################################################
+# delete info user
+#####################################################################
+
+
+@routes.my.get(
+    "/my/user/delete-info",
+    name="my.user.delete-info",
+    summary="Get user delete info",
+    responses=USER_DELETE_INFO_200 | ERROR_401,
+)
+async def get_user_delete_info(request: Request) -> UserDeleteInfoSerializer:
+    """
+    Get some info before deleting a user.
+
+    This info is showed in a modal (before deleting) and have:
```

Comment:
I wouldn't reference to the front-end's modal in the description of the endpoint. Front-end is just one of the many potential users that may use this api.

Reply:
I thought that it is a very specific endpoint and need a explanation of where is used it. I will change it.

Reply:
Done!